scripts/makeDatabase.js

```diff
@@ -0,0 +1,110 @@
+/* eslint-disable array-callback-return */
+/* eslint-disable no-unused-vars */
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* eslint-disable no-plusplus */
```

Comment:
fighting the linter i see :(
Suggestions:
1. add `"no-plusplus": "off"` in the .eslintrc (so i++ etc works inside basic for loops). Not sure why it's not covered by all the `recommended`s in `extends`.
2. This file looks like it's just for testing, but if you want to clean it up, you can replace the 2 offending `.map`s with `.forEach`, so you can remove `eslint-disable array-callback-return`
```suggestion
/* eslint-disable no-unused-vars */
/* eslint-disable @typescript-eslint/no-unused-vars */
```

Reply:
yeah.. some of these are real weird. I kept the .map and changed up eslintrc instead.

yeah you can disregard a lot of the stuff in this file. it'll never really be used for development, and theyre kinda just personal scripts im writing to help me generate files and data for seed files